crates/edr_eth/src/remote/cacheable_method_invocation.rs

```diff
@@ -719,6 +751,21 @@ impl Hasher {
         Ok(this)
     }
 
+    fn hash_reward_percentile(self, value: &RewardPercentile) -> Self {
+        const RESOLUTION: f64 = 100.0;
```

Comment:
Does this mean we treat everything smaller than `0.01` as negligible? Can this introduce issues?

Reply:
Yeah, this means that we allow specifying reward percentiles with 0.01% granularity. I think this is fine, because Foundry caches with 0.5% granularity.